src/status_im/contexts/wallet/bridge/bridge_to/view.cljs

```diff
@@ -46,6 +51,7 @@
   (let [network-details  (rf/sub [:wallet/network-details])
         account          (rf/sub [:wallet/current-viewing-account])
         token            (rf/sub [:wallet/wallet-send-token])
+        selected-network (rf/sub [:wallet/send-network])
```

Comment:
This subscription can be moved to ``bridge-token-component

Reply:
Done